playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -29,6 +29,10 @@
       msg: The nfs group must be limited to one host
     when: "{{ (groups[g_nfs_hosts] | default([])) | length > 1 }}"
 
+  - fail:
```

Comment:
Just noting that this matches the previous `fail` conditions in this file for other groups.